.travis.yml

```
@@ -2,19 +2,15 @@ sudo: false
 
 language: node_js
 
+dist: bionic
+
 node_js:
-  - 6.10.2
+  - 10.15.3
```

The node version upgrade is out of scope for this PR

```suggestion
  - 10.14.2
```